content/employment/index.mdx

```diff
@@ -197,16 +198,16 @@ opportunitiesBody: >
   work at SSW are the great culture, the cool client projects, and the cutting
   edge tech stack. "SSW is unlike any place you've ever worked at."
 opportunities:
-  - opportunityRef: content/opportunities/Senior-Test-Analyst.mdx
+  - opportunityRef: content/opportunities/Systems-Administrator.mdx
+  - opportunityRef: content/opportunities/Building-Manager.mdx
+  - opportunityRef: content/opportunities/Marketing-Assistant-and-Copywriter.mdx
```

Comment:
Hey @camillars It was already removed from title... this is just the file name, which is NOT visible to the public. We could also fix that, but I wouldn't recommend since there is no impact for users.